### ISSUE TITLE
Breadcrumbs - Support lang attribute 

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/vets-design-system-documentation#2868]
## Description
Recent work was finished in #2741 and #2742 to update the content-build liquid filter to use `va-breadcrumbs`. During accessibility review on [the pull request](https://github.com/department-of-veterans-affairs/content-build/pull/2095) it was discovered that breadcrumbs with `-esp` on the end of them do not properly detect language. This is due to the missing `lang` attribute on the anchor tags provided by `va-breadcrumbs`. 

## Considerations
- Current breadcrumbs in production that are generated in content-build have the `lang` attribute
- va-breadcrumbs doesn't currently include the `lang` attribute as a prop so it likely needs to
- This could look like setting `lang` to `en` as default but potentially updating through a prop
- [original comment highlighting the issue](https://github.com/department-of-veterans-affairs/content-build/pull/2095#issuecomment-2127464252)

## Tasks
- [ ] update `va-breadcrumbs` so it supports passing in the `lang` attribute

## Acceptance Criteria 
- [ ] Above tasks have been completed
